### ISSUE TITLE
UX - review default file mapping to apps

### ISSUE THREAD

**Issue by tabesin** [fossasia/meilix#111]
- Issue type: Feature request 

### Short description

The meta package should verify that Meilix gets file associations (mimetypes mapping) right, e.g. that a clicked file is opened by the right software. Usually you have a defaults.list on your system, the directory depends on the Desktop Environment, with the ability of overwrite in your user directory. Thus the package should provide for an overwrite config file in the user directory with entries that make sense.

Furthermore, there are different solutions to determine what becomes the default app: e.g. 
1. changing the order of package installations.
2. changing the system default (risky)
3. changing the user default

This issue is for regular review. Unpleasant default file associations are a common usability glitch on Ubuntu based distributions. E.g. images should not open with Libreoffice Draw by default. To close this issue:

- [ ] Make yourself familiar with the mimetype settings and default apps on your own and the current meilix system
- [ ] in particular: Investigate where the config files are located on debian based systems and meilix
- [ ] and read relevant online sources and forum entries of users who struggle with file mappings.
- [ ] Tweak your own system to get familiar. E.g. if you have acrobat reader installed you probably don't want pdfs to open with an also installed Evice Reader by default.
- [ ] Review if the current system-wide mimetype 2 app mapping on meilix provides for good usability
- [ ] Document your research on file associations mapping in a blog post.
- [ ] Make sure that the metapackage includes a userspace config of file mappings
- [ ] Make sure that the mapped default apps of Meilix match the project specification  

### Usecase

- [ ] Usability test case: create a collection of (freely distributable) test files in different common file formats (not within the repo!) which can be used by a test user to check which app opens a file.

**Comment by xeon-zolt** [comment]
yes like currently no editor is present in the system so libre writer is used by default

**Comment by tabesin** [comment]
For setting:
xdg-mime default application mimetype(s)
For querying:
xdg-mime query default mimetype(s)

**Comment by vaibhavsingh97** [comment]
@tabesin Can i try to solve this? Any resources you want me to go through before trying to solve this issue?

**Comment by tabesin** [comment]
@vaibhavsingh97 See ticket #96.
